### DISCUSSION TITLE
Sorting references / abstracts in Endnote during pre-phase

### DISCUSSION THREAD

**Discussion by Grienknien** [asreview/asreview#700] [Q&A] [answered]
Hello,

I found out about ASReview through a colleague and I am really intrigued by the idea of doing a systematic search with the help of AI. I am still at the phase of pre-screening, so I haven't even really started using ASReview itself. 
In the instruction documentation it is mentioned that it is advised to retrieve as many abstracts as possible after exporting a search to your search manager (in my case Endnote). 
What I'm struggling with now is how I identify the references that MISS an abstract, without the need of having to go through all the 4000 different references. Is there a button, just like title or author, that can categorize the references on having an abstract yes or no?

Kind regards,

Jesse

**Comment by SagevdBrand** [comment, marked as answer]
Hi Jesse,

Thanks for posting your question!

I found the following in EndNote to be helpful:
In the screenshot below you can see that I have added the column Abstract to the general overview. You add this by right clicking this area and then find the Abstract button (quite far below). Once added, you can then press this button to have EndNote sort your references on the abstracts. Those without abstracts should appear at either the top or the bottom. 

![image](https://user-images.githubusercontent.com/64579032/130041708-84353aba-c5f1-4359-998d-0e9320fdb0ca.png)

Let me know if this solved your issue!

**Comment by Grienknien** [comment]
Hi Sofie,

thank you so much, this helped me on the way!

**Comment by boer0107-zz** [comment]
Hi Jesse,

Another option in Endnote is to work with Smart Groups.
Create a smart group and choose the field abstract, choose the option Is (in stead of contains) and leave the box for the value empty. This will return all references for which the abstract field is empty.

**Reply by boer0107-zz** [reply]
Smart groups work better then sorting when you want to filter on multiple values. For instance when you want to make a selection of all references without an abstract AND/OR without a DOI.
